app/models/settings/smtp.rb

```diff
@@ -11,6 +11,10 @@ class SMTP < Base
     setting :password, type: :string, default: ApplicationConfig["SMTP_PASSWORD"].presence
     setting :port, type: :integer, default: ApplicationConfig["SMTP_PORT"].presence || 25
     setting :user_name, type: :string, default: ApplicationConfig["SMTP_USER_NAME"].presence
+    setting :from_email_address, type: :string, default: ApplicationConfig["DEFAULT_EMAIL"].presence,
```

Comment:
I had to have a long hard think about this:

The reason I set the default value to `ApplicationConfig["DEFAULT_EMAIL"]` is because in the case of a Forem Cloud Client, we set this to noreply@forem.com when we provision it. In the case of a self hoster, adding a value for this variable is part of the self host process. So we can assume that in most cases this value will be set and users will most likely want to use the value. 

It does mean that when a Forem Cloud client wants to override their Sengrid account and instead provide their own SMTP settings, that it will pre-populate the reply_to and send_from fields in the custom SMTP form to noreply@forem.com, and they will need to override that values.

Reply:
I also make doubly sure that we're using the correct value [here](https://github.com/forem/forem/pull/16499/files#diff-0cde289ff1ab7adbee93472f177bc29bde9e6091d65c13691e4795238daf0858R25)